features/FEATURE_UVISOR/includes/uvisor/api/inc/uvisor_deprecation.h

```diff
@@ -18,7 +18,8 @@
 #define __UVISOR_DEPRECATION_H__
 
 #if defined(UVISOR_PRESENT) && UVISOR_PRESENT == 1
-#warning "Warning: You are using FEATURE_UVISOR, which is unsupported as of Mbed OS 5.9."
+#warning "Warning: uVisor is superseded by the Secure Partition Manager (SPM) defined in the ARM Platform Security Architecture (PSA). \
+          uVisor is deprecated as of Mbed OS 5.9, and being replaced by a native PSA-compliant implementation of SPM."
```

Comment:
I think this should say 5.10. uvisor is still supported on the 5.9 branch so saying it is deprecated as of 5.9 is misleading.

Reply:
Okay, then @ndevillard needs to reapprove the text with either 5.9 or 5.10

Reply:
please just comment the message that you would like and let's finish with this annoyance

Reply:
@ndevillard - I believe the agreement was deprecation notice in 5.9, actual deprecation in 5.10.  So the notice should indicate 5.10.

Reply:
@Oren i think the 5.9 just needs to change to 5.10 and the rest is fine

Reply:
@adbridge Done.